src/bosh-director/lib/bosh/director/api/controllers/events_controller.rb

```diff
@@ -6,6 +6,19 @@ module Api::Controllers
     class EventsController < BaseController
       EVENT_LIMIT = 200
 
+      get '/:id' do
+        content_type(:json)
+
+        event_id = params[:id]
+        event = Models::Event[event_id]
```

Comment:
@cppforlife it works as is. Do I need to update the code to support event ids as integer?